common/common.c

```diff
@@ -128,7 +102,7 @@ char *bt_common_get_home_plugin_path(void)
 	char *path = NULL;
 	const char *home_dir;
 
-	home_dir = get_home_dir();
+	home_dir = g_get_home_dir();
```

Comment:
Do you get the real user's home directory with `g_get_home_dir()`, even if the process is int setuid/setgid?